highfidelity/hifi
# PAL: Per-Avatar Volume Slider; visual tweaks

This PR adds a volume slider beneath each user's VU meter in the PAL. Using this slider, users can adjust the volume of each avatar present in a domain individually.

*Test Plan*
* In a domain with 1 or more other avatars, make another avatar speak. Ensure that audio stream works as expected.
* Open the PAL, select the speaking avatar, and move the slider associated with the speaking avatar all the way to the right. Ensure that the perceived loudness of that avatar's speech increases. Also notice that the VU meter visualization can now fill the entire width of the line.
* Move the slider associated with the speaking avatar all the way to the left. Ensure that you can no longer hear the speaking avatar. Their VU meter should no longer appear (it should stay invisible until you move the slider some amount to the right).
* Close the PAL and ensure you still cannot hear the speaking avatar.
* Reopen the PAL. Ensure the volume slider for that avatar is still all the way to the left and that their VU meter is still not visible.
* Force the speaking avatar to rejoin the domain. The speaking avatar should be audible at a normal volume again.

This PR also tweaks the color of the VU meter.
